invenio_app_ils/records_relations/api.py

```diff
@@ -148,7 +148,7 @@ def _build_child(self, parent_pid, child_pid, relation_type):
         r.update(metadata or {})
 
         # add also the title of the parent
-        r["title"] = parent.get("title", {}).get("title", "")
+        r["title"] = parent.get("title", {})
```

Comment:
For relations, we just use the title for display purposes so we don't need the entire `title` object so I don't think this change is needed. But would be good to get a confirmation from @ntarocco regarding if we need more than just `title.title`.